lib/uuid_map.rb

```diff
@@ -34,4 +34,18 @@ def rewrite(data)
   def source
     [@newfile, @oldfile].find(&:exist?)
   end
+
+  def remap(from, to)
+    @data ||= mapping
```

Comment:
I'm not following why you're creating a new instance variable for this, especially as it doesn't seem to be used from any other method. Is there some reason why that's necessary/useful, or should this just be a local variable to this method?